### PR TITLE
Major refactoring of types

### PR DESCRIPTION
**Pull request by young-k**
The project is currently architected as follows:
```
src 
|-- schema.py
|-- data.py
```

`schema.py` was bloated with both resolvers and all types, so I changed the architecture to be the following:

```
src 
|-- schema.py
|-- data.py
|-- types/
     |-- __init__.py
     |-- account_info_type.py
     |-- eatery_type.py
     |-- food_station_type.py
     |-- operating_hours_type.py
     |-- payment_methods_type.py
```

I wanted to avoid having all the classes in one file, while avoiding having a file per each class (since each is only a few lines), so I tried to group by functionality.

I also verified that no functionality broke after this refactor.

### CODE REVIEW

**Comment by young-k** [comment]
Rebased & addressed linting comments

File: src/data.py [hunk 1]
```diff
@@ -46,7 +50,7 @@ def parse_eatery(data_json):
     phone = eatery.get('contactPhone', 'N/A')
     phone = phone if phone else 'N/A'  # handle None values
```
Comment (kq-li) [inline src/data.py:51]: `phone or 'N/A'` since `phone` shouldn't be falsy anyway

Comment (kq-li) [reply src/data.py:51]: not your code but just something i noticed

File: src/types/account_info_type.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+from graphene import List, String, ObjectType
+
+class TransactionType(ObjectType):
+  name = String(required=True)
+  timestamp = String(required=True)
+
+class AccountInfoType(ObjectType):
+  brbs = String(required=True)
+  cityBucks = String(required=True)
```
Comment (kq-li) [inline src/types/account_info_type.py]: `city_bucks`

File: src/schema.py [hunk 3]
```diff
@@ -1,107 +1,36 @@
...
-  timestamp = String(required=True)
-
-class AccountInfoType(ObjectType):
-  brbs = String(required=True)
-  cityBucks = String(required=True)
-  history = List(TransactionType, required=True)
-  laundry = String(required=True)
-  swipes = String(required=True)
+  def update_data(eateries):
+    Data.eateries = eateries
```
Comment (kq-li) [inline src/schema.py:21]: no more events or menus?

**Review by kq-li — COMMENTED** [review]
good changes -- a few comments

File: src/schema.py [hunk 4]
```diff
@@ -1,107 +1,36 @@
 from datetime import datetime
 from dateutil import parser
 
-from graphene import Field, ObjectType, String, List, Int, Float, Boolean
-from graphene.types.datetime import Date, Time
+from graphene import Boolean, Field, Int, List, String, ObjectType
```
Comment (meganle) [inline src/schema.py]: alphabetize

File: src/types/account_info_type.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+from graphene import List, String, ObjectType
```
Comment (meganle) [inline src/types/account_info_type.py]: same here

File: src/types/eatery_type.py [hunk 6]
```diff
@@ -0,0 +1,28 @@
+from graphene import Field, Float, Int, List, ObjectType, String
```
Comment (meganle) [inline src/types/eatery_type.py:1]: same

File: src/schema.py (hunk 3 above)
Comment (henrytso) [reply src/schema.py]: I think events (`EventType`) got moved to `types/operating_hours_type.py` and menus (`List(FoodStationType)`) got moved to `types/food_station_type.py`.

**Review by henrytso — COMMENTED** [review]
Looks much cleaner

File: src/schema.py (hunk 3 above)
Comment (young-k) [reply src/schema.py:21]: they're never queried, everything is from `eateries`

File: src/types/account_info_type.py (hunk 2 above)
Comment (young-k) [reply src/types/account_info_type.py]: 😠 i didn't write this, but i'll make the change

File: src/data.py (hunk 1 above)
Comment (young-k) [reply src/data.py:51]: phone can be `None` if in the dictionary `contactPhone: None`

Comment (kq-li) [reply src/data.py:51]: yeah i meant falsy but not `None`, the `or` will handle `None`s properly so it's just slightly cleaner

**Comment by young-k** [comment]
@kq-li @meganle @henrytso addressed comments

**Review by kq-li — APPROVED** [review]
lgtm

**Pull request merged**